icechunk/src/session.rs

```diff
@@ -240,6 +242,26 @@ impl Session {
             .map(|node| self.change_set.update_array(node.id, metadata))
     }
 
+    pub async fn array_has_modified_chunks(
```

Comment:
the name of this function confuses me, because it makes me think this returns true if the session (`self`) has modified any chunks in the array, but I  don't think that's what the function does.

Reply:
function is now deleted.